src/vm/arm64/stubs.cpp

```diff
@@ -2173,10 +2179,49 @@ PCODE DynamicHelpers::CreateDictionaryLookupHelper(LoaderAllocator * pAllocator,
             p += 4;
         }
         
+        if (i == 0 && pLookup->indirectFirstOffset)
+        {
+            // 12 bytes
+            // str x9, [sp, #-0x10]
+            *(DWORD*)p = 0xf81f03e9;
+            p += 4;
+
+            // sub sp, sp, 0x10
+            *(DWORD*)p = 0xd10043ff;
+            p += 4;
```

Comment:
This is an unnecessary instruction.  `str x9, [sp, #-0x10]!` form should be used above.